cpp/demo/biharmonic/main.cpp

```diff
@@ -1,124 +1,160 @@
-// Biharmonic equation (C++)
-// =========================
+// ---
+// jupyter:
+//   jupytext:
+//     cell_metadata_filter: -all
```

Comment:
What does this do?

Reply:
It turns out it isnt needed unless for connecting the scripts to notebooks.